operations/customers.md

```diff
@@ -652,7 +652,7 @@ Attaches the specified file to the customer profile. Note this operation support
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
 | `CustomerId` | string | required | Unique identifier of the [Customer](#customer). |
-| `EnterpriseId` | string | optional | Unique identifier of the [Enterprise](enterprises.md#enterprise). Required when using a [Portfolio Access Token](../guidelines/multi-property.md), ignored otherwise. |
+| `ChainId` | string | optional | Unique identifier of the chain. Required when using [Portfolio Access Tokens](../guidelines/multi-property.md), ignored otherwise. |
```

Comment:
Kariem caught this mistake so this is a fix for previous pr

Reply:
Fix the changelog as well